LegendaryExplorer/LegendaryExplorer/Tools/PackageEditor/Experiments/PackageEditorExperimentsS.cs

```diff
@@ -2690,7 +2690,7 @@ public static void DumpSound(PackageEditorWindow packEd)
                         using IMEPackage pcc = MEPackageHandler.OpenMEPackage(filePath);
                         foreach (ExportEntry export in pcc.Exports.Where(exp => exp.ClassName == "WwiseStream"))
                         {
-                            if (export.ObjectNameString.Split(',') is string[] { Length: > 1 } parts && parts[0] == "en_us" || parts[0] == "en-us" && parts[1] == tag)
+                            if (export.ObjectNameString.Split(',') is string[] { Length: > 1 } parts && parts[0] == "en_us" | parts[0] == "en-us" && parts[1] == tag)
```

Comment:
You will want to use ||, not |. | is for bitwise operations, we are doing boolean logic, so we use ||.